### ISSUE TITLE
Can you explain the meaning of your datasets?

### ISSUE THREAD

**Issue by chenchongthu** [guyulongcs/SIGIR2020_NICF#1]
E.g., which dataset it is (Movielens, EachMovie, or Netflix?), and how to split it into training set and test set as in your sigir paper?

**Comment by anirudhgj** [comment]
+1 
It would be really helpfull , if you could kindly explain data processing.

**Comment by guyulongcs** [comment]
For the details of data processing, please send your email to Dr. Lixin Zou (zoulixin15@gmail.com).
